src/overlays/actors/ovl_En_Box/z_en_box.c

```diff
@@ -28,66 +68,651 @@ const ActorInit En_Box_InitVars = {
     (ActorFunc)EnBox_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80869B50[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_U8(targetMode, 0, ICHAIN_STOP),
 };
 
-#endif
-
-extern InitChainEntry D_80869B50[];
-
-extern UNK_TYPE D_0600043C;
-extern UNK_TYPE D_06000DB0;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Box/func_80867BD0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Box/func_80867BDC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Box/func_80867C14.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Box/func_80867C8C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Box/func_80867FBC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Box/func_80867FE4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Box/func_8086800C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Box/EnBox_Init.s")
+typedef struct {
+    f32 data[5];
+} EnBox_PlaybackSpeed; // 0x14
```

Comment:
Does this need to be an struct?
This seems to be just an array of floats

Reply:
In `EnBox_WaitOpen` there appears to be a struct copy that prevents me from using an array